go.mod

```diff
@@ -62,7 +64,7 @@ replace (
 
 require (
 	github.com/antihax/optional v1.0.0
-	github.com/rancher/rancher/pkg/apis v0.0.0-20241127174121-c051d99dcded
+	github.com/rancher/rancher/pkg/apis v0.0.0
```

Comment:
@slickwarren What’s the reason to not pin to a specific commit?

Reply:
it was causing problems when importing harvester, I believe because it also has a pinned version: https://github.com/harvester/harvester/blob/master/go.mod#L15